package.json

```diff
@@ -43,5 +43,9 @@
         "pretty-quick": "^3.1.3",
         "ts-jest": "^27.1.4",
         "ts-node-dev": "^1.1.8"
-    }
+    },
+    "engines": {
+        "node": ">=16.6.0"        
+    },
+    "engineStrict": true
```

Comment:
Is this necessary? What impacts does it have?

Reply:
It makes sure that the bot cannot run when using node < 16.6 which will break the bot